database/migrations/platform/2024_05_21_000000_update_siteawards_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('SiteAwards', function (Blueprint $table) {
+            $table->dropColumn('User');
+        });
```

Comment:
I'm confused:
```
SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'User' doesn't exist in table (Connection: mysql, SQL: alter table `SiteAwards` drop `User`)
```
```
MariaDB [retroachievements-web]> desc SiteAwards;
+----------------+---------------------+------+-----+---------+----------------+
| Field          | Type                | Null | Key | Default | Extra          |
+----------------+---------------------+------+-----+---------+----------------+
| id             | bigint(20) unsigned | NO   | PRI | NULL    | auto_increment |
| AwardDate      | datetime            | NO   |     | NULL    |                |
| User           | varchar(50)         | NO   | MUL | NULL    |                |

```

Reply:
This came down to the unique constraint we used to have before the table actually had a primary key. Fixed in latest.